### PR TITLE
fixes issues with the separate modsuit part activation

### LINKED ISSUE #87413: Taking/placing items in modsuit storage spams override runtimes for `mousedropped_onto` over itself
### Client Version:

_No response_

### Issue Summary:

```
[16:25:56] Runtime in code/datums/signals.dm, line 43: mousedropped_onto overridden. Use override = TRUE to suppress this warning. Target: the bluespace rapid part exchange device (/obj/item/storage/part_replacer/bluespace/tier4) Existing Proc: mousedrop_receive New Proc: mousedrop_receive
proc name: stack trace (/proc/_stack_trace)
usr: Sealed101/(Tzeentch Who-Was-Light)
usr.loc: (Supermatter Engine Room (114,74,2))
src: null
call stack:
stack trace("mousedropped_onto overridden. ...", "code/datums/signals.dm", 43)
/datum/storage (/datum/storage): RegisterSignal(the bluespace rapid part excha... (/obj/item/storage/part_replacer/bluespace/tier4), "mousedropped_onto", "mousedrop_receive", 0)
/datum/storage (/datum/storage): attempt insert(the bluespace rapid part excha... (/obj/item/storage/part_replacer/bluespace/tier4), Tzeentch Who-Was-Light (/mob/living/carbon/human), 0, 0, 1)
the storage (/atom/movable/screen/storage/cell): Click(null, "mapwindow.map", "icon-x=10;icon-y=23;left=1;but...")
Sealed101 (/client): Click(the storage (/atom/movable/screen/storage/cell), null, "mapwindow.map", "icon-x=10;icon-y=23;left=1;but...")
```

### Round ID:

_No response_

### Test Merge Information:

_No response_

### Reproduction Steps:

take/place an item from/into modsuit storage

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

generally cleans up the code a bit. fixes the issue where if you had clothing on yourself already and you tried extending when active it still went through the sealing process even though it didnt extend the part, causing weird desyncs. fixes the issues with part enabled modules that would not activate, makes stealth and radproof modules require the whole suit to be out cause thinking about it they wouldnt really work without that i think
reverts quick activation to try put parts on you instead of removing them as priority, i think that plays nicer with separate part activation
fixes #87413

## Changelog

:cl:
fix: fixes some issues with separate modsuit part activation
fix: rad protection, miner and stealth modsuit modules require the full suit out
fix: armor booster now applies slowdown removal per part instead of to the control unit
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
what this needs now is just fixes for sprites hopefully and smartkars working on that

**Comment by DaydreamIQ** [comment]
Might wanna add armor booster to the list of things you need the whole suit for cause right now I've seen people use it without the suit deployed to run at meth speed

**Comment by Fikou** [comment]
i also fix that in this pr it doesnt need full extension it just needs to have fixed sprites

**Comment by Fikou** [comment]
breaks something with storage i think

File: code/modules/mod/modules/modules_engineering.dm [hunk 1]
```diff
@@ -244,13 +244,13 @@
 	target.AddComponent(/datum/component/tether, src, 7, "tether")
 
 /datum/embed_data/tether_projectile
-	embed_chance=65 // spiky
-	fall_chance=2
+	embed_chance = 65 // spiky
+	fall_chance = 2
 	ignore_throwspeed_threshold=TRUE
```
Comment (MrMelbert) [inline code/modules/mod/modules/modules_engineering.dm:249]:
```suggestion
	ignore_throwspeed_threshold = TRUE
```

File: code/modules/mod/modules/modules_engineering.dm [hunk 2]
```diff
@@ -244,13 +244,13 @@
...
 
 /datum/embed_data/tether_projectile
-	embed_chance=65 // spiky
-	fall_chance=2
-	ignore_throwspeed_threshold=TRUE
-	pain_stam_pct=0.4
-	pain_mult=3
-	jostle_pain_mult=2
-	rip_time=1 SECONDS
+	embed_chance = 65 // spiky
```
Comment (Fikou) [inline code/modules/mod/modules/modules_engineering.dm:247]:
```suggestion
	embed_chance = 65 //spiky
```

**Pull request merged**